portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -11,72 +11,68 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
+ 
 package com.google.sps.servlets;
 
-import java.io.IOException;
+import static java.util.Objects.isNull;
+
 import com.google.appengine.api.datastore.DatastoreService;
 import com.google.appengine.api.datastore.DatastoreServiceFactory;
 import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
+import java.io.IOException;
+import java.util.List;
+import java.util.ArrayList;
```

Comment:
So close! Just alphabetize (`ArrayList` before `List`).

FYI: setting up linting will catch this sort of thing for you. Here's a guide Chibuzo and Anthony followed; I'd recommend setting it up as soon as you're able: https://github.com/g-luo/step/blob/master/linting.md